app.html

```diff
@@ -1025,11 +1036,15 @@
 }
 
 async function Run2() {
-  document.getElementById('while_connected').style.visibility='visible';  
-  document.getElementById('while_disconnected').style.visibility='hidden';  
+//  document.getElementById('while_connected').style.visibility='visible';  
```

Comment:
Looks like ehse lines were commented out and moved further down.
Is there a reason for that?
If so, why leave the commented out lines?

Reply:
This is to be able to load all data, show 'connecting' while the initialization of all the HTML elements is going on, and make the page visible when all presets, tracks and fonts are retrieved.
It also helps for BLE connections that fail to show that the code 'hangs'. Otherwise elements are shown but are not actually usable because the connection is not ready or in the process of failing. 
The comments is a note-to-self were these lines originally were implemented. Will remove them.